Detections/Okta_CL/LoginfromUsersfromDifferentCountrieswithin3hours.yaml

```diff
@@ -0,0 +1,27 @@
+id: 2954d424-f786-4677-9ffc-c24c44c6e7d5
+name: Login from User(s) from Different Countries within 3 hours
+description: |
+  'This creates an incident in the event that a user logs into the Okta Console from different countries within 3 hours'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: OktaSSO
+    dataTypes: 
+      - Okta_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitalAccess 
+relevantTechniques:
+  - T1078
+query: |
+
+  let timeframe = ago(3h);
+  Okta_CL
+  | where published_t >= timeframe
+  | where eventType_s == "user.session.start"
+  | where outcome_result_s == "SUCCESS"
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), NumOfCountries = dcount(client_geographicalContext_country_s) by actor_alternateId_s
+  | where NumOfCountries > 2
```

Comment:
may want to consider making this a parameter via - let threshold = 2; 
and modify to - | where NumOfCountries > threshold

Reply:
threshold added as a parameter